source/locates.html.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Locates FIX API Specification
+
+
+toc_footers:
+  - [Powered By Slate](https://github.com/slatedocs/slate)
+
+search: true
+---
+
+# Introduction
+Locates FIX API specification details the tags and values along with description of values for each field for processing locate requests and acceptances from OMS clients. Clear Street currently utilizes FIX 4.2 format.
+
+# Configuring FIX Session
+```
+// Example Config
+[DEFAULT]
+ReconnectInterval=5
+SenderCompID=LOCATESCLIENT
+HeartBtInt=30
+SocketAcceptPort=31234
+TargetCompID=CLST
+HeartBtInt=30
+SocketConnectPort=31234
+SocketConnectHost=100.0.0.0
+BeginString=FIX.4.2
+
+[SESSION]
+StartTime=04:00:00
+EndTime=22:00:00
+```
+A FIX session is defined as a unique combination of a `BeginString` (the FIX version number 4.2), a `SenderCompID` (OMS Client ID as defined by Clear Street), and a `TargetCompID`.
+
+A FIX session has an active time period during which all data is transmitted. Clear Street currently accepts connections between 4 AM EST to 10 PM EST on all business days. Clear Street will act as an acceptor and OMS will be the initiator.
+
+A `Logon` message (`Tag 35=A`) must be sent to Clear Street on every business day to indicate the begining of activity and a `Logout` message (`Tag 35=5`)must be sent to Clear Street indicating end of activity for that day. 
+
+Clear Street also recommends exchange of heartbeats every 30 seconds by sending a message with `Tag 35=0`.
+
+Clear Street expects every message sent to have a unique continuous sequence number as part of message with Tag 34=. If there are any gaps in sequence numbers, a sequence reset message will sent to OMS client with tag `34=4`.
+
+# Message List
+
+| Message | Tag 35 | Comments | Direction w.r.t. CLST |
+| --- | --- | --- | --- | --- |
+| New Order - Single | D | 1. Make a single locate request 2. Accept/Reject a single locate | Incoming |
+| New Order - List | E | 1. Make multiple locate requests 2. Accept/Reject multiple locates | Incoming |
+| Execution Report | 8 | 1. Offer response for a single locate request with locate id 2. Status response for a single locate accept/reject message with locate id 3. Multiple status responses for a list of locate accept/reject messages with locate ids | Outgoing |
+| Reject | 3 | 1. Any validation/authentication errors on the Order requests | Outgoing |
+| Quote Request | R | 1. Request a quote for a list of securities | Incoming |
+| Quote | S | 1. Response for quote request. Multiple responses are send depending on the number of quotes requested. | Outgoing |
+
+# New Order - Single (Request)
+
+```
+// Example 1: By Symbol
+8=FIX.4.2\u00019=0054\u000135=D\u000149=OMSC\u000156=CLSTLOCT\u000134=1360\u000152=20220208-21:10:01\u000111=12345671\u000155=IBM\u000154=1\u000160=20220208-21:10:00.792\u000140=1\u0001109=ACCOUNTID\u000176=USRNM,PSWD\u000138=1000\u0001440=TRADERID\u00011=ACCOUNTID\u000158=THISISASINGLELOCATEREQUESTBYSYMBOL\u000110=106\u0001
```

Comment:
Possibly a gap in my understanding but I've always seen FIX as XML... potentially this is more correct but if it is usually sent in XML then I think there would be value in breaking out the tags to make it approachable and readable

Reply:
FIX format is not an XML. I think the confusion could be that OCC uses FIXML which is different from FIX. Please check the existing FIX trade spec. https://clear-street.github.io/FIX-docs/#introduction